sphinx_gallery/directives.py

```diff
@@ -11,11 +12,26 @@
 from sphinx.errors import ExtensionError
 
 
+from .backreferences import (
+    _thumbnail_div,
+    THUMBNAIL_PARENT_DIV,
+    THUMBNAIL_PARENT_DIV_CLOSE,
+)
+from .gen_rst import extract_intro_and_title
+from .py_source_parser import split_code_and_text_blocks
+
+
 class MiniGallery(Directive):
     """Custom directive to insert a mini-gallery.
 
-    The required argument is one or more fully qualified names of objects,
-    separated by spaces.  The mini-gallery will be the subset of gallery
+    The required argument is a one or more of the following:
```

Comment:
Think this is required for RST to render properly
```suggestion
    The required argument is a one or more of the following:

```
and https://output.circle-artifacts.com/output/job/ff1728af-0451-4fef-bab5-967229bb60df/artifacts/0/html/gen_modules/sphinx_gallery.directives.html#sphinx_gallery.directives.MiniGallery seems to suggest this as well